modules/plugins/wfly-10/src/main/resources/META-INF/rhq-plugin.xml

```diff
@@ -8981,11 +8982,12 @@
             <c:list-property name="connectors" required="false" description="Specifies the names of connectors that will be broadcast." >
               <c:simple-property name="connectors" />
             </c:list-property>
-            <c:simple-property name="socket-binding" required="false" type="string" readOnly="false" description="The broadcast group socket binding. Alternative to jgroups-channel">
+            <c:simple-property name="socket-binding" required="false" type="string" readOnly="false" description="The broadcast group socket binding. Alternative to Jgroups Channel. Since EAP 7.2 is an alternative to Jgroups Cluster">
```

Comment:
I think the description requires some work. ``Since EAP 7.2 is an`` .. I don't think the EAP 7.2 is the alternative.

Reply:
I propose for consistency:

```xml
            <c:simple-property name="socket-binding" required="false" type="string" readOnly="false" description="The broadcast group socket binding. Prior to EAP 7.2: Alternative to jgroups-channel. EAP 7.2+: Alternative to jgroups-cluster.">
```